lite-rpc/src/bridge.rs

```diff
@@ -334,4 +335,23 @@ impl LiteRpcServer for LiteBridge {
             Ok(None)
         }
     }
+
+    async fn get_epoch_info(
+        &self,
+        _config: Option<RpcContextConfig>,
+    ) -> crate::rpc::Result<EpochInfo> {
+        let block_info = self
+            .data_cache
+            .block_information_store
+            .get_latest_block_info(CommitmentConfig::finalized())
```

Comment:
Here commitment should come from the config

Reply:
Ah yes, you're right and it's not the right commitment.